documents/knowledge/elements.yaml

```diff
@@ -2059,8 +2059,335 @@
     - "[420, 69] : 4"
     - '["abc", 10] : "   abc    "'
     - '["no", "yesnt"] : " no "'
+- element: "⋎"
+  name: Bitwise Or
+  arity: 2
+  description: Performs bitwise or between two numbers / Removes a character at nth index / Merges strings on longest common prefix and suffix
+  overloads:
+    num-num: a | b
+    num-str: b[:a]+b[a+1:]
+    str-num: a[:b]+a[b+1:]
+    str-str: merge_join(a,b)
+  vectorise: true
+  tests:
+    - "[420, 69] : 485"
+    - '[2, "abc"] : "ab"'
+    - '["abc", 2] : "ab"'
+    - '["Hello", "lower"] : "Hellower"'
+- element: "꘍"
+  name: Bitwise Xor
+  arity: 2
+  description: Performs bitwise xor between two numbers / appends n spaces to a string / prepends n characters to a string / Levenshtein Distance
+  overloads:
+    num-num: a ^ b
+    num-str: \" \" * a + b
+    str-num: a + \" \" * b
+    str-str: levenshtein_distance(a,b)
+  vectorise: true
+  tests:
+    - "[420, 69] : 481"
+    - '[5, "ab"] : "     ab"'
+    - '["ab", 5] : "ab     "'
+    - '["atoll", "bowl"] : 3'
+- element: "ꜝ"
+  name: Bitwise Not
+  arity: 1
+  description: Performs bitwise not on a number / check if any letters are uppercase
+  overloads:
+    num: ~a
+    str: any_upper(a)
+  vectorise: true
+  tests:
+    - "[220] : -221"
+    - '["Hello"] : 1'
+- element: "℅"
+  name: Random Choice
+  arity: 1
+  description: Random choice of single item from array
+  overloads:
+    lst: random.choice(a)
+  vectorise: false
+  tests:
+    - "[[1,2,3]] : 2"
+    - "[[1,2,3]] : 1"
+    - "[[1,2,3]] : 3"
+- element: "≤"
+  name: Lesser Than or Equal To
+  arity: 2
+  description: a is lesser than or equal to b?
+  overloads:
+    any-any: a <= b
+  vectorise: true
+  tests:
+    - "[1,2] : 1"
+- element: "≥"
+  name: Greater Than or Equal To
+  arity: 2
+  description: a is greater than or equal to b?
+  overloads:
+    any-any: a >= b
+  vectorise: true
+  tests:
+    - "[1,2] : 0"
+- element: "≠"
+  name: Not Equal To
+  arity: 2
+  description: a is not equal to b?
+  overloads:
+    any-any: a != b
+  vectorise: true
+  tests:
+    - "[1,2] : 1"
+- element: "⁼"
+  name: Exactly Equal To
+  arity: 2
+  description: a equal to b? (non-vectorizing)
+  overloads:
+    any-any: a == b
+  vectorise: true
```

Comment:
bruh

Reply:
it literally says non-vectorising in the description

Reply:
uhh sory bout that